util/chplenv/chpl_llvm.py

```diff
@@ -117,6 +117,11 @@ def check_llvm_packages(llvm_config):
     usr_include_clang_ok = False
 
     include_dir = run_command([llvm_config, '--includedir']).strip()
+    if "/nix/store" in include_dir:
```

Comment:
I think this could work OK, but would it be better to have `include_dir.startswith("/nix/store")` ?